packages/@sanity/field/src/diff/components/DiffErrorBoundary.tsx

```diff
@@ -39,17 +39,25 @@ export class DiffErrorBoundary extends React.Component<BoundaryProps, BoundarySt
 
     const help = isDev ? <p>Check the developer console for more information.</p> : null
     return (
-      <>
-        <div className={styles.root}>
-          <span className={styles.icon}>
-            <ErrorOutlineIcon />
-          </span>
-          <div className={styles.message}>
-            The component responsible for showing the changes to this field has crashed.
-            {help}
-          </div>
-        </div>
-      </>
+      <Card tone="critical" padding={3} paddingBottom={0}>
+        <Flex align="flex-start">
+          <Box>
+            <Text>
+              <Icon symbol="error-outline" />
```

Comment:
We should import the `ErrorOutlineIcon` directly, to avoid bundling all the icons 😬

Reply:
Ok!